sp800-63c/sec5_assertions.md

```diff
@@ -2,7 +2,7 @@
 
 ## 5. Assertions
 
-An assertion is a packaged set of attribute values and/or attribute claims about an authenticated subscriber that is passed from the identity provider (IdP) to the relying party (RP) in a federated identity system. Assertions contain a variety of information, including assertion metadata, attribute values and claims about the subscriber, and other information that the RP can leverage, such as restrictions, expiration time, etc.
+An assertion is a packaged set of attribute values or claims about or associated with an authenticated subscriber that is passed from the identity provider (IdP) to the relying party (RP) in a federated identity system. Assertions contain a variety of information, including assertion metadata, attribute values and claims about the subscriber, and other information that the RP can leverage, such as restrictions, and expiration time.
```

Comment:
restore the term "attribute claims" instead of "claims" (to be replaced by another term such as "attribute reference" later, but that's a separate edit).